cli/src/cmd/test.js

```diff
@@ -141,6 +142,10 @@ export async function testApp({
 
   // run the temp image
   spinner.start('Running app docker image...\n');
+  const taskTimeoutWarning = setTimeout(() => {
+    spinner.warn('Task is taking longer than expected...');
+    spinner.start('Running app docker image...\n');
```

Comment:
Reset previous spinner text rather than duplicating the text for easier maintenance

```suggestion
    const spinnerText = spinner.text;
    spinner.warn('Task is taking longer than expected...');
    spinner.start(spinnerText); // restart spinning
```